ketcher-autotests/tests/Structure-Creating-&-Editing/Text-Tool/text-formating.spec.ts

```diff
@@ -0,0 +1,254 @@
+    await selectTopPanelButton(TopPanelButton.Copy, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await page.keyboard.press('Control+v');
+    await page.getByTestId('canvas').click({ position: { x: 250, y: 300 } });
+    await takeEditorScreenshot(page);
+  });
+
+  test('Text tool - Checking if user is able to cut and paste the created text objects', async ({
+    page,
+  }) => {
+    await openFileAndAddToCanvas('KET/text-EPMLSOPKET-2272.ket', page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await selectNestedTool(page, SelectTool.LASSO_SELECTION);
+    await createSomeStructure(page);
+    await selectTopPanelButton(TopPanelButton.Cut, page);
+    await page.keyboard.press('Control+v');
+    await clickInTheMiddleOfTheScreen(page);
+    await takeEditorScreenshot(page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+  });
+
+  test('Text tool - Selection of different types of text objects', async ({
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-2274
+    // Checking if its possible to select a text objects of any size by clicking on green frame
+    await openFileAndAddToCanvas('KET/text-file-EPMLSOPKET-2274.ket', page);
+    await page.getByText('TEXT').dblclick();
+    await page.getByRole('dialog').getByRole('textbox').click();
+    await page.keyboard.press('Control+a');
+    await page
+      .getByRole('dialog')
+      .getByRole('textbox')
+      .fill(
+        'PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP',
+      );
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await page.keyboard.press('Control+a');
+    await page
+      .getByText(
+        'PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP',
+      )
+      .dblclick();
+    await page.keyboard.press('Control+a');
+    await page.keyboard.press('Delete');
+    await pressButton(page, 'Apply');
+    await openFromFileViaTextBox('tests/test-data/Txt/longtext_test.txt', page);
+    await clickInTheMiddleOfTheScreen(page);
+  });
+
+  test("Text tool - UTF-8 compatible ('Paste from Clipboard')", async ({
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-5253
+    // Checking if possible is add UTF-8 data format  to canvas
+    await selectTopPanelButton(TopPanelButton.Open, page);
+    await openFromFileViaClipboard(
+      'tests/test-data/KET/utf-8-svg-png.ket',
+      page,
+    );
+    await clickInTheMiddleOfTheScreen(page);
+  });
+});
+
+export async function openFromFileViaTextBox(filename: string, page: Page) {
+  const fileText = await readFileContents(filename);
+  await page.getByTestId('text').click();
+  await clickInTheMiddleOfTheScreen(page);
+  await page.getByRole('dialog').getByRole('textbox').fill(fileText);
+  await waitForLoad(page, () => {
+    pressButton(page, 'Apply');
+  });
+}
```

Comment:
Functions should be at the top of the file after imports. And they don’t write 'export' before 'async'